toolset/run-tests.py

```diff
@@ -115,9 +115,9 @@ def main(argv=None):
     if defaults['server_host'] is None:
         defaults['server_host'] = defaults['client_host']
 
-    maxThreads = 8
+    defaults['threads'] = 8
     try:
-        maxThreads = multiprocessing.cpu_count()
+        defaults['threads'] = multiprocessing.cpu_count() * 2
```

Comment:
Doesn't this count the number of cpus on the `application` server and not the `load` server (where it is used)?